packages/create-react/src/components/widgets/FormulaWidget.tsx

```diff
@@ -45,7 +49,9 @@ export function FormulaWidget({
         }),
       )
       .then((response) => {
-        setValue(response.value);
+        if (response.value) {
+          setValue(response.value);
+        }
```

Comment:
Minor: In the Angular example I think this was solved with `setValue(response.value ?? 0)`, either is OK with me but probably we want both to work the same way!

If we think that the source should throw an error if no tiles are loaded yet, so this will fall into the catch block, we could change the API client to do that as well.

Reply:
Yes, I agree these widgets should have uniform behaviour